superflore/parser.py

```diff
@@ -16,19 +16,20 @@
 
 
 # set up a parser and return it
-def get_parser(tool_tip, is_generator=True):
+def get_parser(tool_tip, is_generator=True, exclude_all=False):
```

Comment:
@andre-rosa Why is this option being removed?

Reply:
Hi @allenh1  << That's to enforce generating recipes for a single ROS_DISTRO at a time for OpenEmbedded. Then there'll be separate PR-s for each ROS distro with their description adjusted specifically for the one being generated.